src/components/library-item/library-item.css

```diff
@@ -3,8 +3,9 @@
 
 .library-item {
     display: flex;
+    align-self: stretch;
     flex-direction: column;
-    align-items: center;
+    align-items: left;
```

Comment:
This causes the text on for example the sound library to be left aligned:
![image](https://user-images.githubusercontent.com/654102/48425957-d2283980-e733-11e8-8235-d3917875d098.png)

Reply:
Oh got it, will try to address with CSS.